AirCasting/SearchAndFollow/SearchMap/SearchMapView.swift

```diff
@@ -16,21 +16,39 @@ struct SearchMapView: View {
     }
     
     var body: some View {
-        ZStack(alignment: .top, content: {
-            LoadingView(isShowing: $viewModel.showLoadingIndicator, activityIndicatorText: Strings.SearchMapView.loadingText) {
-                map
-            }
-            VStack(alignment: .center, content: {
-                addressTextField
-                measurementTypeText
-                searchAgainButton
-                // This is temporary untill we implement clickable cards
-                Spacer()
-                Button(action: { showCompleteScreen = true }, label: { Text("Complete screen") })
+        GeometryReader { reader in
+            ZStack(alignment: .top, content: {
+                LoadingView(isShowing: $viewModel.showLoadingIndicator, activityIndicatorText: Strings.SearchMapView.loadingText) {
+                    ZStack(alignment: .top) {
+                        map
+                        VStack(alignment: .leading) {
+                            Spacer()
+                            cardsTitle
+                            cards
+                                .frame(width: reader.size.width, height: reader.size.height / 7, alignment: .leading)
+                        }
+                        .padding(.horizontal, 5)
+                        .padding(.bottom, 5)
+                    }
+                }
+                VStack(alignment: .center, content: {
+                    addressTextField
+                    measurementTypeText
```

Comment:
The measurement type text is still "particular matter" instead of "particulate matter". Could you change it in this PR before we forget to do it?

Reply:
For now here it's just a mock, but I will change not to mess up anything. Thanks!